.npmignore

```diff
@@ -3,5 +3,6 @@
 !README.md
 !bin/*
 !lib/**/*
+!downloads/*
```

Comment:
I think the intention of this file was to ignore everything (the `*` at the top) and the selectively whitelist only certain files from existing in the repo (with the exclamation prefix).

Maybe * is only matching files in the root and we actually need **/* ?

Reply:
ah! can make that change